.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
```

Comment:
yes it is, but avocado is now for python2, so that we cannot move to higher version now. it is planned for future.